README.md

```diff
@@ -1,5 +1,6 @@
-# pandas-mentoring
-<img src="img/logo.jpeg" width="600" height="200"><br>
+# Mentoring new pandas contributors (team 2019)
+
+This is the list of the [GitHub](https://github.com) usernames of all the people interested in contributing to [pandas](https://github.com/pandas-dev/pandas):
```

Comment:
I guess this was from previous work, seems unrelated to this PR.

Happy to get proposals for the README, but if you want to do, better in a separate PR.

Reply:
@datapythonista sorry and yes, the ```README``` change was from previous work (I should not have added it in my PR). Now I submitted a new PR (#40) which only contains the change in the ```environment.yml```